ocs_ci/ocs/node.py

```diff
@@ -301,6 +301,52 @@ def add_new_node_and_label_it(machineset_name):
     return new_spun_node[0]
 
 
+def add_new_node_and_label_upi(node_type, num_nodes, mark_for_ocs_label=True):
+    """
+    Add a new node for aws upi platform and label it
```

Comment:
Need to change the description ( aws ). see https://github.com/red-hat-storage/ocs-ci/pull/1847/files#diff-ffa18f30e526ae7b2498579992b406efR320